tests/test_pages.py

```diff
@@ -114,6 +116,16 @@ def test_opening_magic_link_with_a_next_sets_a_new_url(client):
     assert response.url == "/test_dashboard/?a=test&b=test"
 
 
+def test_opening_magic_link_with_a_unsafe_next_sets_triggers_404(client):
+    token = factories.MagicTokenFactory()
+    url = (
+        reverse("magicauth-validate-token", kwargs={"key": token.key})
+        + "?next=http://www.myfishingsite.com/?a=test&b=test"
+    )
+    response = client.get(url)
+    assert response.status_code == 404
+
```

Comment:
A tester aussi, eventuellement : 
 - le user n'est pas logged in
 - le token est tjs valide (on devrait peut etre l'invalider d'ailleurs, non?)

Reply:
Peut-être rajouter un test : l'utilisateur est déjà loggé, il clique un lien de fishing. L'utilisateur reste loggé et reçoit une 404